migration/sql-files/072-adds-active-flag-in-iteration.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE iterations ADD COLUMN user_active bool DEFAULT false NOT NULL;
```

Comment:
To be safe, you *can* update every existing iteration in order to have the `user_active` set to `false`:

```sql
UPDATE iterations SET user_active = false;
```
**Update:** You said on Mattermost, that you have prove, that this is not needed.

Reply:
@kwk yes